backend/lib/geocoder.py

```diff
@@ -55,8 +49,11 @@ def geocode_batch(self, addresses):
         The results are returned in the same order as the original list.
         """
         try:
-            raise Exception('want to switch to single')
             results = self._safe_geocode(addresses)
+        except GeocodioAuthError as error:
+            logger.exception(error)
```

Comment:
do we definitely want to logger exception? it prints the stack trace and can be kinda confusing. If we know what the error is and don't need to debug, we can keep to logging.error?

Reply:
Done